### DISCUSSION TITLE
Reinforce mining error > JSONDecodeError: Expecting value

### DISCUSSION THREAD

**Discussion by litoturtle** [coccoinomane/crabada.py#102] [Q&A] [answered]
Hello all,

When I run the command to reinforce mining I am getting this issue once in a while. The first crab would rent from tavern then this error below would show up. Any ideas on how to fix?

Thanks

![image](https://user-images.githubusercontent.com/104463321/167282127-eba0d18c-5643-4638-953b-9fd416046b56.png)

**Comment by litoturtle** [comment, marked as answer]
Fix, seems like the api call is getting rejected for 100 line items. Just had to decrease the query.

**Reply by coccoinomane** [reply]
Thank you for sharing your solution, @litoturtle !

I have added the error message to the discussion's title so that others might benefit from your answer.

Cheers,
Cocco
